.github/test-spec.yml

```diff
@@ -599,3 +599,20 @@
   - "samples/nrf_rpc/protocols_serialization/**/*"
   - "subsys/net/openthread/rpc/**/*"
   - "subsys/nrf_rpc/**/*"
+
+"CI-secdom-test":
+  - "samples/bluetooth/multiple_adv_sets/**/*"
+  - "samples/nfc/record_text/**/*"
```

Comment:
Why SecDom CI should be run on NFC and applications changes?

Reply:
We are on a later revision of sdk-nrf and want to test our newest release with samples built from latest sdk-nrf.

Reply:
> Why SecDom CI should be run on NFC and applications changes?

That sample is used to test peripheral access configuration via secdom.